src/dispatch/signal/models.py

```diff
@@ -311,6 +312,7 @@ class SignalBase(DispatchBase):
     project: ProjectRead
     source: Optional[SourceBase]
     variant: Optional[str]
+    lifecycle: Optional[str]
```

Comment:
Should we set a default lifecycle here? Are we hoping to restrict the possible lifecycles? If so an `enum` might be warranted.

Reply:
Since this will be propagated from Geiger where we already have an enum to restrict possible values and the value in Dispatch should be readonly I'm not sure it's necessary.